test/LP/LP.jl

```diff
@@ -40,29 +40,39 @@ const LP_TEST_FILE = "test.lp"
             "Binary\n" *
             "x\n" *
             "End\n"
+
```

Comment:
Maybe here `@test !MOI.is_empty(model)` so that we check it `MOI.empty!` is actually doing what it is supposed to.